src/components/composite/layout/App/App.tsx

```diff
@@ -27,6 +28,7 @@ import Main from './Main';
 const App = ({ children }): React.ReactElement => {
   const { t } = useTranslation();
   const { isAuthenticated } = useAuthenticationContext();
+  useUpdateSubscription();
```

Comment:
Is it correct to have the update subscription at the App level?

Reply:
Yes, we want to be notified about all community updates at all times. I assumed that this subscription is global and should be on a root level in the app.